src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -208,6 +238,41 @@ impl DeviceManager {
         Ok(device_id)
     }
 
+    async fn create_vhost_blk_device(
+        &mut self,
+        config: &VhostUserConfig,
+        device_id: String,
+    ) -> Result<ArcMutexDevice> {
+        let mut vhu_blk_config = config.clone();
+        // get hypervisor block driver
+        let block_driver = match self
```

Comment:
I think you could store the `block_driver` in the device manager to reduce the duplicate code, as `create_block_device` will do this operation again.

Reply:
thx. fixed